resources/views/searches/tasks/index.blade.php

```diff
@@ -1,5 +1,64 @@
 <!-- Content - OPEN -->
 <div class="container margin-top padding-bottom">
-    <img src="/img/mockup_tasks.png" style="width: 100%" />
+	@include('searches.tasks.task_table')
+
+	<div class="container padding-top">
+        <div class="row align-items-start">
+            <div class="col">
+        		<div class="text-left">
+                    <!-- Add Task - OPEN -->
+                    <a href="{{ Route('createTask', $search->id) }}" class="btn btn-lg btn-outline-primary margin-left align-right btn-sm margin-bottom" role="button"
+                    <?php if ($search->status == 1 || Auth::user()->profile == 'guest'){ ?> style="display: none" <?php } ?> >
+                        <span class="octicon octicon-plus"></span>
+                        {{ __('actions.create') . ' ' . __('main.task') }}
+                    </a>
+                    <!-- Add Task - CLOSE -->
+        		</div>
+            </div>
+        <!-- Select filter -->
+        <div class="col">
+            <div class="text-right">
+                <!-- Status -->
+                <div class="row">
+                    <div class="col">
+                <select id="statusFilter" class="form-control">
+                    <option value="">
+                        {{ __('activity.every_status') }}
+                    </option>
+                    <option>{{ __('activity.done') }}</option>
+                    <option>{{ __('activity.in_progress') }}</option>
+                    <option>{{ __('activity.to_do') }}</option>
+                </select>
```

Comment:
Each option must have a value attribute. If not it will not send any value to the backend so anything is save for the task